tools/js-sdk-release-tools/package.json

```diff
@@ -9,9 +9,10 @@
     "prepack": "npm run build"
   },
   "bin": {
-    "changelog-tool": "./dist/generateChangelogAndBumpVersionCLI.js",
-    "hlc-code-gen": "./dist/generateCodesInLocalCLI.js",
-    "hlc-code-gen-for-pipeline": "./dist/generateCodesInPipelineCLI.js"
+    "changelog-tool": "./dist/changelogTool.js",
+    "hlc-code-gen": "./dist/hlcCodeGenCli.js",
+    "hlc-code-gen-for-pipeline": "./dist/hlcCodegenForPipelineCli.js",
+    "rlc-code-gen": "./dist/llcCodegenAutomationCLI.js"
```

Comment:
why there's no rlc-code-gen-for-pipeline ?

Reply:
Because the tool doesn't support running in pipeline now.